src/test/java/FizzBuzzTest.java

```diff
@@ -0,0 +1,44 @@
+import com.tcs.java8.utility.FizzBuzzUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.util.StringUtils;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class FizzBuzzTest {
+
+    @Test
+    public void shouldReturnFizzWhenInputNumberDividedBy3() {
```

Comment:
method name does not give proper requirement as 15 is divisible by 3 also.
shouldReturnFizzWhenInputNumnerDivisibleBy3AndNotBy15